doc/api/url.md

```diff
@@ -1558,11 +1558,12 @@ A `TypeError` is thrown if `urlString` is not a string.
 
 A `URIError` is thrown if the `auth` property is present but cannot be decoded.
 
-Use of the legacy `url.parse()` method is discouraged. Users should
-use the WHATWG `URL` API. Because the `url.parse()` method uses a
-lenient, non-standard algorithm for parsing URL strings, security
-issues can be introduced. Specifically, issues with [host name spoofing][] and
-incorrect handling of usernames and passwords have been identified.
+Use of the legacy `url.parse()` method is not encouraged as it is not officially
+supported by the project. Users are recommended to use the WHATWG `URL` API
+instead. Because the `url.parse()` method uses a lenient, non-standard algorithm
+for parsing URL strings, security issues can be introduced. Specifically, issues
+with [host name spoofing][] and incorrect handling of usernames and passwords
+have been identified.
```

Comment:
This seems to make the text longer but doesn't actually clarify anything. If anything, it makes it less clear.

Before: This API is discouraged. Use this other API instead.

After: This API is not encouraged. (OK, but is it discouraged? Or just not encouraged?) It is not officially supported by the project. (OK, but is it unofficially supported? What is the difference between officially supported and supported otherwise? And if it's not supported by the project, then is it supported by other people?) 

In my opinion, the added context makes things less clear, not more clear.

If we're looking to clarify the text, here is my suggestion:

```suggestion
Avoid `url.parse()` if you can. Use the WHATWG `URL` API instead. 
`url.parse()` uses a lenient, non-standard algorithm for parsing URL
strings. It is prone to security issues such as [host name spoofing][]
and incorrect handling of usernames and passwords.
```